tools/datagen/Cargo.toml

```diff
@@ -39,7 +39,7 @@ icu_testdata = { version = "0.3", path = "../../provider/testdata", features = [
 log = "0.4"
 reqwest = { version = "0.11", features = ["json", "stream", "gzip"] }
 serde = { version = "1.0", default-features = false, features = ["derive", "alloc"] }
-simple_logger = "1.11"
+simple_logger = "1.12"
```

Comment:
Nit: Also please look in the docs and Makefile.toml files and see if we can remove the -v from anywhere.